.github/workflows/CD.yml

```
@@ -0,0 +1,40 @@
+name: CD
+
+on:
+  push:
+    branches:
+        - master
```

```suggestion
        - main
```
Changed to the `main` branch.
Also the default branch for this repository is `develop` not `main`, so the default pull request target is `develop` also - you might have to change the target!

---

@jo-basevi pointed out we should be deployed based on tagging, which I agree with, so this is now moot, but thanks for spotting.